speech/grpc/transcribe_streaming.py

```diff
@@ -64,7 +64,7 @@ def make_channel(host, port):
     return implementations.secure_channel(host, port, composite_channel)
 
 
-def _audio_data_generator(buff):
+def _audio_data_generator(buff, stoprequest):
```

Comment:
Hm... sure, okay, I can see that. I think there's a case for just letting `_fill_buffer` propagate the way it is, but I won't block on it.